### DISCUSSION TITLE
建议开发deepin版的“玩机技巧”

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#3986] [General | 常规讨论]
对于刚接触deepin的用户，“帮助手册”内容太过庞杂机械，“欢迎”又太简单

不妨借鉴 鸿蒙手机系统 的 玩机技巧 的模式，按使用场景，将deepin的常用功能，以图文和视频的形式进行说明演示

对于其实很好用，但知道的人不多的功能，也借此进行宣介

我觉得，这会大大提高用户的上手速度

欢迎大家讨论

**Comment by BLumia** [comment]
独立成一个应用提供玩机技巧的内容？是的话或许 deepin-home 深度之家适合放这种板块？ cc @myml 

以及如果觉得文档内容值得改善的话，其实是欢迎你直接参与到文档的完善之中的。我们有个 dde-doc-doc-go 文档小组，可以加入进来参与到 SIG 的讨论之中 :)

**Reply by linuxlionking** [reply]
如果您使用过鸿蒙的 玩机技巧 app ，应该能明白我想表达的核心意思  
展示的技巧，不宜多，但足够高频使用 ，贴近用户使用场景
可以理解为“欢迎” 的进阶版功能

**Reply by linuxlionking** [reply]
帮助手册和wiki ，并不适合刚接触系统的用户
不是说咱的文档或手册写的不好，而是不适合小白新手这些受众

**Reply by BLumia** [reply]
确实没有用过鸿蒙。cc @meiyixiang 

但按你的回复所表达的，应该确实是个独立应用吧。

**Reply by fslong520** [reply]
如果玩过网游的都知道有个新手村，有个游戏引导，引导你怎么放技能，怎么使用游戏里的各种功能。
我觉得开发一个这样的app对新手会很有帮助。

**Reply by BLumia** [reply]
理论上说，帮助手册也是给新手看的。从我的角度看，是在考虑这种内容应当出现在哪里合适。

另外提供面向新手的文档也是文档完善的一部分，所以如果要出单独的玩机技巧应用的话，我们其实也鼓励社区来一同开发这种玩机技巧的应用。

**Reply by linuxlionking** [reply]
恕我直言，帮助手册，就是系统功能的机械罗列  各家都一样，手册就是要这样组织内容
但别说新手了，我这老手都看不下去帮助手册
可以拿身边同事的鸿蒙手机，体验一下 玩机技巧 app ，您就能理解我想表达的了
这个就应该官方来做，社区搞，无法保证质量

**Reply by BLumia** [reply]
我不反对加这样的内容呀，我的意思是：

> 从我的角度看，是在考虑这种内容应当出现在哪里合适。

意思是深度之家里增加对应的模块还是独立的应用的形式，亦或是帮助手册新增一个快速上手模块之类。

> 社区搞，无法保证质量

但我不认同这种观点。深度社区的发展意义就在于让所有真正希望改善它的人都能参与进来。如果需要保障质量的方法，就由大家一起集思广益来拟定与执行。

**Reply by linuxlionking** [reply]
关于质量这块，各自保留意见
关于以何种方式呈现，我说说自己看法：当前系统已经有 “欢迎” 这个软件了，就在这里呈现
用户装完系统，首次登录，就弹出，文案后面再说，让用户知道，他想快速上手系统，有这么个功能模块

**Comment by xzl01** [comment]
deepin doc doc go SIG就在做这个内容，入门的手册需要有人情味，提供信息的效率可以低一点，不过要更适合小白阅读

**Reply by linuxlionking** [reply]
对，目的就是让刚接触deepin的新手，快速熟悉系统
不能是简单的功能罗列，而是更场景化，小白化
